Cargo.toml

```diff
@@ -140,28 +140,48 @@ secp256k1 = { version = "0.27", default-features = false, features = [
 celestia-rpc = { git = "https://github.com/eigerco/lumina" }
 celestia-types = { git = "https://github.com/eigerco/lumina" }
 
-alloy = { git = "https://github.com/alloy-rs/alloy.git", package = "alloy", features = [
+alloy = { git = "https://github.com/alloy-rs/alloy.git", package = "alloy", rev = "607359a8d7ba7543d702da5d406b7bc78eff3d9f", features = [
     "pubsub",
     "signers",
     "signer-wallet",
     "signer-yubihsm",
 ] }
 alloy-primitives = { version = "0.7.2", default-features = false }
-alloy_provider = { git = "https://github.com/alloy-rs/alloy.git", package = "alloy-provider", features = [
+alloy_provider = { git = "https://github.com/alloy-rs/alloy.git", package = "alloy-provider", rev = "607359a8d7ba7543d702da5d406b7bc78eff3d9f", features = [
```

Comment:
Since we keep `Cargo.lock` under version control, the purpose of this is to protect against inadvertent updates, correct?

Reply:
Yes, I ran into that an hour back. We got the latest upstream which broke our code.